docs/content/docs/install/manual_demo/_index.md

```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
-- `bookwarehouse` application that receives requests from `bookstore` to restock books
+In this section we will deploy 4 different Pods, and we will apply policies to control the traffic between them.
 
-When we demonstrate traffic splitting using SMI Traffic Split, we will deploy an additional application:
+- `bookbuyer` is an HTTP client makeing requests to `bookstore`. This traffic is **permitted**.
+- `bookthief` is an HTTP client and much like `bookbuyer` also makes HTTP requests to `bookstore`. This traffic should be **blocked**.
+- `bookstore` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
+- `bookwarehouse` is a server and should respond only to `bookstore`. Both `bookbuyer` and `bookthief` should be blocked.
 
-- `bookstore-v2` application representing version v2 of the `bookstore` application
 
-The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` demo applications will be installed in their respective Kubernetes Namespaces. For OSM to work, each application Pod must contain an Envoy proxy as a sidecar container. OSM can automatically inject an Envoy proxy sidecar into application Pods. Use the `osm` CLI to add Kubernetes Namespaces to monitor and inject proxies into using the `osm namespace add` command. Once the Namespace is added, OSM will monitor the Namespace for new Pods and automatically inject the Envoy proxy as a sidecar into each Pod created in the Namespace.
+We are going to craft SMI policies, which will bring us to this final desired
+state of allowed and blocked traffic between pods:
 
-### Create the Bookstore Application Namespaces
+| from  /   to: | bookbuyer | bookthief | bookstore | bookwarehouse |
+|---------------|-----------|-----------|-----------|---------------|
+| bookbuyer     |     \     |     ❌     |     ✔    |       ❌       |
+| bookthief     |     ❌     |     \     |     ❌     |       ❌       |
+| bookstore     |     ❌     |     ❌     |     \     |       ✔      |
+| bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
+
+
+To show SMI Traffic Split, we will deploy an additional application:
+
+- `bookstore-v2` - a server, which is a newer version of the existing `bookstore` application
+
+The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces with
+the same names. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
+
+### Create the Namespaces
 
 ```bash
-for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; done
+kubectl create namespace bookstore
+kubectl create namespace bookbuyer
+kubectl create namespace bookthief
+kubectl create namespace bookwarehouse
 ```
 
-### Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
+### Join the new namespaces to the service mesh
```

Comment:
small nit/preference: "Add the new namespaces to the OSM control plane"

feel free to ignore if you don't agree.

Reply:
@michelleN I agree and I very much appreciate that.
I always think of `join namespace` and it sounds awkward (going to stop this going forward).
`Onboard` sounds too complicated.

`Add namespace` is simple and unambiguous!
Thank you!